src/domain/app/__tests__/App.test.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import App from '../App';
+import { shallow } from 'enzyme';
+
+describe('App', () => {
+  it('renders without crashing', () => {
+    expect(shallow(<App />)).toMatchSnapshot();
```

Comment:
The snapshot here doesn't really reveal much about the component, it took a snapshot for the shallow wrapper itself.
```suggestion
    expect(shallow(<App />).html()).toMatchSnapshot();
```

Reply:
Fixed.